src/usr/lib/rsetup/cli/system.sh

```diff
@@ -118,7 +118,7 @@ update_hostname() {
     hostnamectl set-hostname "$hostname"
     cat << EOF > "/etc/hosts"
 127.0.0.1 localhost
-127.0.1.1 $(hostnamectl hostname)
+127.0.1.1 $(hostnamectl hostname 2>/dev/null || hostname)
```

Comment:
```suggestion
127.0.1.1 $(hostnamectl hostname || echo $hostname)
```
错误消息不会被`$()`捕获，所以不需要重定向。刚刚只是为了给你展示
另外后面的变量要echo出去

Reply:
不echo的那个是命令，我们在构建系统的时候在容器里面跑，得到的hostname不一定正确